odoo/tests/common.py

```diff
@@ -415,17 +415,20 @@ def with_user(self, login):
     @contextmanager
     def _assertRaises(self, exception, *, msg=None):
         """ Context manager that clears the environment upon failure. """
-        with super(BaseCase, self).assertRaises(exception, msg=msg) as cm:
+        with ExitStack() as init:
```

Comment:
What about `outer` (in response to `inner`) or `init_stack` ?

Reply:
Its purpose is not to be the "outer" stack, which is why it then gets gutted, it's the stack to handle the initialisation phase. Hence init.